src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-list/cephfs-subvolume-list.component.ts

```diff
@@ -125,7 +125,7 @@ export class CephfsSubvolumeListComponent extends CdForm implements OnInit, OnCh
       {
         name: $localize`Path`,
         prop: 'info.path',
-        flexGrow: 1,
+        flexGrow: 0.5,
```

Comment:
what are you trying to achieve with this? If I'm not mistaken this is a prop from the old datatable that does nothing in the carbon table.

Reply:
Ah okay, makes sense. I was trying to reduce the column size for one of the row to make space for other rows.. The path cell is too big..